packages/ava-react/src/NarrativeTextVis/paragraph/Heading.tsx

```diff
@@ -1,20 +1,25 @@
 import React from 'react';
-import { HeadingParagraphSpec, getHeadingWeight } from '@antv/narrative-text-schema';
+
+import { getHeadingWeight } from '@antv/ava';
 import { isNaN } from 'lodash';
+
 import * as Elements from '../styled';
 import { Phrases } from '../phrases';
 import { getPrefixCls, classnames as cx } from '../utils';
-import { ExtensionProps, PhraseEvents } from '../interface';
 import { presetPluginManager } from '../chore/plugin';
 
+import type { NtvTypes } from '@antv/ava';
+import type { ExtensionProps, PhraseEvents } from '../types';
+
 type HeadingProps = ExtensionProps &
   PhraseEvents & {
-    spec: HeadingParagraphSpec;
+    spec: NtvTypes.HeadingParagraphSpec;
   };
 
 export function Heading({ spec, pluginManager = presetPluginManager, ...events }: HeadingProps) {
   const weight = getHeadingWeight(spec.type);
   if (isNaN(weight)) return null;
+  // eslint-disable-next-line import/namespace
```

Comment:
Elements includes H1 ~ H6. Selection by variable name is not allowed here, so I disable it.